tests/milestone1_test_cases.md

```diff
@@ -0,0 +1,44 @@
+FIMS testing plan for milestone 1 is documented using the test case documentation template from the [FIMS developer Handbook](https://noaa-fims.github.io/collaborative_workflow/testing.html#test-case-documentation-examples).
+
+## General test case documentation
+
+The test case documentation below is a general case to apply to many functions/modules. For individual functions/modules, please make detailed test cases for specific options, noting "same as the general test case" where appropriate.
+
+- Test ID: General test case
+- Feature to be tested
+    - The function/module returns correct output values given different input values
+    - The function/module returns error messages when users give wrong types of inputs
+    - The function/module notifies an error if the input value is outside the bound of the input parameter
+
+- Approach
+    - Prepare expected true values using R
+    - Run tests in R using testthat and compare output values with expected values
+    - Push tests to the working repository and run tests using GitHub Actions
+    - Submit pull request for code review
+
+- Evaluation criteria
+    - The tests pass if the output values equal to the expected true values
```

Comment:
You might want to make it clear to compare within a tolerance such that the values don't have to be exactly equal.

Reply:
Thanks for the suggestion. I added some clarifications to the evaluation criteria.